### PR TITLE
Update for PureScript 0.14

### PR DESCRIPTION
**Pull request by jhbertra**
- [x] refactor nix to use https://github.com/input-output-hk/haskell.nix, give some useful dev tools in shell
- [x] add support for lorri
- [x] update src/Language/PureScript/Bridge/Printer.hs to import genericShow from correct location for PS 0.14

### CODE REVIEW

**Review by hrajchert — APPROVED** [review]
LGTM, you might want to ask @shlevy for a review as well as I don't have enough nix knowledge, and also, if we merge right now, how would that affect the different projects that depend on this? You mentioned that you were able to compile marlowe run to use this, but  should we update all sites at the same time?

**Comment by jhbertra** [comment]
It would have no effect - we pin a specific revision in the main repo

File: shell.nix [hunk 1]
```diff
@@ -1,30 +1,10 @@
-{ nixpkgs ? import <nixpkgs> {}, compiler ? "default" }:
+{ nixpkgs ? import <nixpkgs> {} }:
```
Comment (shlevy) [inline shell.nix:1]: This is no longer needed

**Review by shlevy — APPROVED** [review]
One nit but LGTM!

File: shell.nix (hunk 1 above)
Comment (jhbertra) [reply shell.nix:1]: :+1:

**Pull request merged**